upgrade/UPGRADE-v9.0.0-dev.md

```diff
@@ -87,6 +87,13 @@ There you can find links to upgrade notes for other versions too.
           +    message: '#^Property (Shopsys|Tests)\\.+::\$.+ \(.+\) does not accept (object|object\|null)\.$#'
           +    path: %currentWorkingDirectory%/tests/FrontendApiBundle/*
       ```
+- add optional fronted API - images to your project ([#1486](https://github.com/shopsys/shopsys/pull/1486))
```

Comment:
```suggestion
- add optional fronted API - products' and categories' images to your project ([#1486](https://github.com/shopsys/shopsys/pull/1486))
```
I guess there will be much more types of images

Reply:
done